doc/languages.rst

```diff
@@ -902,6 +902,15 @@ to view the docstring of the syntax checker.  Likewise, you may use
          `flycheck-scalastylerc` is not set or the configuration file not found
          this syntax checker will not be applied.
 
+.. supported-language:: Scheme
+
+   Flycheck checks CHICKEN Scheme files with `csc`.
```

Comment:
You need to backticks on either side to mark `csc` as literal.  With just one backtick it's a cross-reference.